### PR TITLE
Added susy to zen image

### PR DESCRIPTION
**Pull request by hog-seruj**

### CODE REVIEW

File: susy/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM mhart/alpine-node:7
+
+MAINTAINER Sergiy Borodulin
```
Comment (andypost) [inline susy/Dockerfile]: https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

File: susy/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+VOLUME /work
+WORKDIR /work
+
+ENV NPM_PACKAGES "susy"
```
Comment (andypost) [inline susy/Dockerfile]: It should be "`$NPM_PACKAGES` susy"

File: susy/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+RUN apk add --no-cache make g++ python \
+  && npm install -g \
+  gulp \
+  gulp-sass \
+  && apk del --purge make g++ python
```
Comment (andypost) [inline susy/Dockerfile]: This done in latest image, just add packages

**Review by andypost — CHANGES_REQUESTED** [review]
needs polishing

**Review by andypost — APPROVED** [review]

File: susy/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,16 @@
+FROM skilldlabs/frontend:zen-next
+
+ARG BUILD_DATE
+ARG VCS_REF
+
+LABEL org.label-schema.build-date=$BUILD_DATE \
+  org.label-schema.vcs-ref=$VCS_REF \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.name="zen-frontend-env" \
+  org.label-schema.description="Zen theme for Drupal - npm, gulp, bower, suzy & libsass" \
```
Comment (pabloguerino) [inline susy/Dockerfile:10]: There is a typo in `susy`

Comment (andypost) [reply susy/Dockerfile:10]: thanx fixed!

Comment (andypost) [reply susy/Dockerfile:10]: I mean in https://github.com/skilld-labs/docker-frontend/pull/20

**Pull request merged**